wfe/wfe.go

```diff
@@ -987,128 +988,18 @@ func (wfe *WebFrontEndImpl) RevokeCertificate(ctx context.Context, logEvent *web
 	}
 }
 
-func (wfe *WebFrontEndImpl) logCsr(request *http.Request, cr core.CertificateRequest, regID int64) {
-	var csrLog = struct {
-		ClientAddr string
-		CSR        string
-		Requester  int64
-	}{
-		ClientAddr: web.GetClientAddr(request),
-		CSR:        hex.EncodeToString(cr.Bytes),
-		Requester:  regID,
-	}
-	wfe.log.AuditObject("Certificate request", csrLog)
-}
-
 // NewCertificate is used by clients to request the issuance of a cert for an
 // authorized identifier.
 func (wfe *WebFrontEndImpl) NewCertificate(ctx context.Context, logEvent *web.RequestEvent, response http.ResponseWriter, request *http.Request) {
-	body, _, reg, prob := wfe.verifyPOST(ctx, logEvent, request, true, core.ResourceNewCert)
+	_, _, _, prob := wfe.verifyPOST(ctx, logEvent, request, true, core.ResourceNewAuthz)
```

Comment:
This is strange. Why both continuing to verify the POST, and why change the expected resource to "new-authz"? I think we should remove this and unconditionally send the "not supported" error. After all, this endpoint gets no traffic now.

Reply:
Oops, this is a copy-paste error from my previous change which was removing the NewAuthorization codepath. I've removed the validation.